app/Models/Permission.php

```diff
@@ -96,6 +96,8 @@ class Permission extends Model implements Validatable
 
     public const ACTION_SETTINGS_RENAME = 'settings.rename';
 
+    public const ACTION_SETTINGS_DESCRIPTION = 'settings.description';
```

Comment:
New permissions need to be added to the frontend:

https://github.com/pelican-dev/panel/blob/82b0aff105730ce5b8bd410756c72c97c582e14a/app/Filament/Server/Resources/UserResource/Pages/ListUsers.php#L116-L119

https://github.com/pelican-dev/panel/blob/82b0aff105730ce5b8bd410756c72c97c582e14a/app/Filament/Server/Resources/UserResource/Pages/ListUsers.php#L351-L360